UI/adminmenu.html

```diff
@@ -32,15 +32,15 @@ <h3><i>Click on the button to perform task</h3></i>
                             <hr>
                               
 							
-									 <div id="item">
+									 <div id="item" style="background-image:url('assets/images/sandwich.jpg'); background-size:352px 252px;">
 										<p><h3><font color="white">Local Food</h3></p></font color>
 										<a><button id="delete" onclick="alert('Are you sure you want to delete this food item?')">Delete</button> 
```

Comment:
Prefer `CSS` over `HTML` for presentational requirements, so instead of using the `tag` font, you would use CSS `font family`